howso/utilities/feature_attributes/base.py

```diff
@@ -1254,13 +1257,11 @@ def infer_loose_feature_bounds(min_bound: int | float,
     def _get_cont_threshold(self, feature_name: str) -> int:
         """Get the minimum number of unique values a feature must have to be considered continuous."""
         n_cases = self._get_num_cases(feature_name)
-        num_series = 1
-        if getattr(self, 'id_feature_names', None):
-            num_series = self._get_unique_count(self.id_feature_names)
-            # If the provided feature is stationary, we should simply evaluate the number of series
-            if feature_name in self.time_invariant_features:
-                return math.ceil(pow(num_series, 0.5))
-        return math.ceil(pow((n_cases / num_series), 0.5))
+        # If the provided feature is stationary, we should simply evaluate the number of series
+        if getattr(self, 'id_feature_names', None) and feature_name in self.time_invariant_features:
+            return math.ceil(pow(self.num_series, 0.5))
+        # Return the sqrt of max(avg. cases per series, num. series)
+        return math.ceil(pow(max(self.num_series, (n_cases / self.num_series)), 0.5))
```

Comment:
This is perfectly fine, but I just wanted you to know that this exists: https://docs.python.org/3.13/library/math.html#math.sqrt (and will perform slightly faster that `pow(..., 0.5)`).